extensions/vertx-http/runtime/pom.xml

```diff
@@ -100,6 +100,14 @@
             <artifactId>mockito-junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.graalvm.sdk</groupId>
+            <artifactId>nativeimage</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-devtools-utilities</artifactId>
```

Comment:
(This comment was added at the end just so you have the chronology of my thoughts)

I'm not exactly excited about this one. It's supposed to be about devtools utilities and while it only contains two classes for now, I'm not sure it's good idea to use it - it could contain dependencies we don't want  in the future.

So, maybe we should actually tackle the issue of using the `Brotli4jLoader` instead of our own code from the start?

Also, @dmlloyd , I wonder if we should have something about the architecture in SmallRye Common OS - in the vein of what we have here: https://github.com/quarkusio/quarkus/blob/main/independent-projects/tools/utilities/src/main/java/io/quarkus/utilities/OS.java#L35

Finally, please make sure you add the actual dependencies before the test ones. That's a convention we try to use.